<html>
<body>
<h3>Limma Analysis Output:</h3>
Links to PDF copies of plots are in 'Plots' section below <br />
<img src="boxplots.png" alt="Boxplots.png" height="500" width="1000"/>
<img src="mdsscree.png" alt="MDSPlot_WT24LO_KO24LO.png" height="500" width="1000"/>
<img src="mdsplot_extra.png" alt="MDSPlot_extra.png" height="500" width="1000"/>
<img src="voomplot.png" alt="VoomPlot" height="500" width="500"/>
<img src="saplot.png" alt="SAPlot.png" height="500" width="500"/>
<img src="mdvolplot_WT24LO-KO24LO.png" alt="MDVolPlot_WT24LO-KO24LO" height="500" width="1000"/>
<h4>Differential Expression Counts:</h4>
<table border="1" cellpadding="4">
<tr>
<td></td>
<th>Up</th>
<th>Flat</th>
<th>Down</th>
</tr>
<tr>
<th>WT24LO-KO24LO</th>
<td>1066</td>
<td>25378</td>
<td>735</td>
</tr>
</table><h4>Plots:</h4>
<a href="boxplots.pdf" target="_blank">BoxPlots.pdf</a><br />
<a href="mdsscree.pdf" target="_blank">MDSPlot_WT24LO_KO24LO.pdf</a><br />
<a href="mdsplot_extra.pdf" target="_blank">MDSPlot_extra.pdf</a><br />
<a href="mdplots_samples.pdf" target="_blank">MDPlots_Samples.pdf</a><br />
<a href="voomplot.pdf" target="_blank">VoomPlot.pdf</a><br />
<a href="saplot.pdf" target="_blank">SAPlot.pdf</a><br />
<a href="mdplot_WT24LO-KO24LO.pdf" target="_blank">MDPlot_WT24LO-KO24LO.pdf</a><br />
<a href="volplot_WT24LO-KO24LO.pdf" target="_blank">VolcanoPlot_WT24LO-KO24LO.pdf</a><br />
<a href="heatmap_WT24LO-KO24LO.pdf" target="_blank">Heatmap_WT24LO-KO24LO.pdf</a><br />
<a href="stripcharts_WT24LO-KO24LO.pdf" target="_blank">Stripcharts_WT24LO-KO24LO.pdf</a><br />
<h4>Tables:</h4>
<a href="limma-voom_WT24LO-KO24LO.tsv" target="_blank">limma-voom_WT24LO-KO24LO.tsv</a><br />
<h4>Glimma Interactive Results:</h4>
<a href="glimma_MDS/MDS-Plot.html" target="_blank">Glimma_MDSPlot.html</a><br />
<a href="glimma_WT24LO-KO24LO/MD-Plot.html" target="_blank">Glimma_MDPlot_WT24LO-KO24LO.html</a><br />
<a href="glimma_volcano_WT24LO-KO24LO/XY-Plot.html" target="_blank">Glimma_VolcanoPlot_WT24LO-KO24LO.html</a><br />
<p>Alt-click links to download file.</p>
<p>Click floppy disc icon associated history item to download all files.</p>
<p>.tsv files can be viewed in Excel or any spreadsheet program.</p>
<h4>Additional Information</h4>
<ul>
<li>TMM was the method used to normalise library sizes.</li>
<li>The limma-voom method was used.</li>
<li>Weights were not applied to samples.</li>
<li>eBayes was used with robust settings (robust = TRUE).</li>
<li>MD Plot highlighted genes are significant at FDR of 0.05 and exhibit log2-fold-change of at least 0.</li>
</ul>
<h4>Summary of experimental data:</h4>
<p>*CHECK THAT SAMPLES ARE ASSOCIATED WITH CORRECT GROUP(S)*</p>
<table border="1" cellpadding="3">
<tr>
<th>SampleID</th>
<th>WT24LO_KO24LO (Primary Factor)</th>
<tr>
<th>HISAT2.on.KO2.24.LO..aligned.reads..BAM.</th>
<td>KO24LO</td>
</tr>
<tr>
<th>HISAT2.on.KO1.24.LO..aligned.reads..BAM.</th>
<td>KO24LO</td>
</tr>
<tr>
<th>HISAT2.on.WT1.24.LO..aligned.reads..BAM.</th>
<td>WT24LO</td>
</tr>
<tr>
<th>HISAT2.on.WT2.24.LO..aligned.reads..BAM.</th>
<td>WT24LO</td>
</tr>
</table><h3>Citations</h3>
Please cite the following paper for this tool:
<br>
Liu R, Holik AZ, Su S, Jansz N, Chen K, Leong HS, Blewitt ME, Asselin-Labat ML, Smyth GK, Ritchie ME (2015). Why weight?  Modelling sample and observational level variability improves power  in RNA-seq analyses. Nucleic Acids Research, 43(15), e97.
<h4>limma</h4>
Please cite the paper below for the limma software itself. Please also try to cite the appropriate methodology articles that describe the statistical methods implemented in limma, depending on which limma functions you are using. The methodology articles are listed in Section 2.1 of the <a href="http://www.bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf">limma User's Guide</a>. Cite no. 3 only if sample weights were used.
<ol>
<li>Smyth GK (2005). Limma: linear models for microarray data. In: 'Bioinformatics and Computational Biology Solutions using R and Bioconductor'. R. Gentleman, V. Carey, S. doit,. Irizarry, W. Huber (eds), Springer, New York, pages 397-420.</li>
<li>Law CW, Chen Y, Shi W, and Smyth GK (2014). Voom: precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biology 15, R29.</li>
<li>Ritchie ME, Diyagama D, Neilson J, van Laar R, Dobrovic A, Holloway A and Smyth GK (2006). Empirical array quality weights for microarray data. BMC Bioinformatics 7, Article 261.</li>
</ol>
<h4>edgeR</h4>
Please cite the first paper for the software itself and the other papers for the various original statistical methods implemented in edgeR.  See Section 1.2 in the <a href="http://www.bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf">edgeR User's Guide</a> for more detail.
<ol>
<li>Robinson MD, McCarthy DJ and Smyth GK (2010). edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140</li>
<li>Robinson MD and Smyth GK (2007). Moderated statistical tests for assessing differences in tag abundance. Bioinformatics 23, 2881-2887</li>
<li>Robinson MD and Smyth GK (2008). Small-sample estimation of negative binomial dispersion, with applications to SAGE data. Biostatistics, 9, 321-332</li>
<li>McCarthy DJ, Chen Y and Smyth GK (2012). Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation. Nucleic Acids Research 40, 4288-4297</li>
</ol>
<p>Please report problems or suggestions to: su.s@wehi.edu.au</p>
<a href="session_info.txt" target="_blank">Session Info</a><br />
<table border="0">
<tr>
<td>Task started at:</td>
<td>2023-03-16 20:02:07</td>
</tr>
<tr>
<td>Task ended at:</td>
<td>2023-03-16 20:02:56</td>
</tr>
<tr>
<td>Task run time:</td>
<td>49 secs</td>
<tr>
</table>
</body>
</html>